### ISSUE TITLE
Reduce MariaDB size

### ISSUE THREAD

**Issue by edolstra** [NixOS/nixpkgs#7114]
The MariaDB package produces a 261 MB directory `$out/mysql-test`, which contributes quite a bit to the 491 MB total package size.

In the MySQL derivation, we remove this and some other cruft:
```
pkgs/servers/sql/mysql/5.5.x.nix:    rm -r $out/mysql-test $out/sql-bench $out/data
pkgs/servers/sql/mysql/5.5.x.nix:    rm $out/share/man/man1/mysql-test-run.pl.1
```
I guess the same should be done for MariaDB.

@shlevy @thoughtpolice @wkennington

**Comment by wkennington** [comment]
I'm not really sure how that directory is 261MB but we can definitely remove it. The closure size on my system is 155MB with mysql-test taking 51MB.

**Comment by wkennington** [comment]
This should help: 53f1d8147db9af0aaac12deff28544f61dc05c64

**Comment by wkennington** [comment]
I'll look into it.

**Comment by wkennington** [comment]
@peti It should propagate now.

**Comment by peti** [comment]
Very cool. Thank your very much for the effort you've put into this re-factoring!

**Comment by vcunat** [comment]
CC a 14.12 fast fix cf46c88a.

**Comment by zimbatm** [comment]
Closing as the size is now acceptable:
```
$ cd nixpkgs
$ nix-build . -A mariadb
$ du -shL result
135M	result
```
